hub/config/src/main/java/uk/gov/ida/hub/config/domain/TransactionConfig.java

```diff
@@ -65,14 +65,6 @@
     @JsonProperty
     protected List<LevelOfAssurance> levelsOfAssurance;
 
-    @Valid
-    @JsonProperty
```

Comment:
There are transactions using these keys in their config files in verify-hub-federation-config. Is there any danger if things falling over if the key no longer exists here?

Reply:
When I start local hub using this `verify-hub` branch, and using the `ida-hub-acceptance-tests` [test-rp.yml transaction config](https://github.com/alphagov/ida-hub-acceptance-tests/blob/718130a4f950f193b95be2dfd6e33331115aa3c4/configuration/hub/config-service-data/transactions/test-rp.yml#L51) (on master)  it still has the `eidasEnabled=true` flag, and it starts ok.

I'm confident this transaction config file is used because [if I then set `enabled=false`](https://github.com/alphagov/ida-hub-acceptance-tests/blob/718130a4f950f193b95be2dfd6e33331115aa3c4/configuration/hub/config-service-data/transactions/test-rp.yml#L33), then the journey fails in the Hub.